app/models/green_lanes/category_assessment.rb

```diff
@@ -72,6 +72,10 @@ def match?(regulation_id:, measure_type_id:, geographical_area: nil)
           self.geographical_area == GeographicalArea::ERGA_OMNES_ID)
     end
 
+    def excluded_geographical_areas
```

Comment:
Should we add a small test for this method?
(on ../green_lanes/category_assessment_spec.rb)